pos_warning_exiting/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Point Of Sale - Warning on Exiting",
+    "summary": "Add warning at exiting the PoS front office UI"
+    " if there are pending draft orders",
+    "version": "12.0.1.0.0",
+    "category": "Point Of Sale",
+    "author": "GRAP, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
+    "license": "AGPL-3",
+    "depends": [
+        "point_of_sale",
+    ],
+    "data": [
+        "views/templates.xml",
+    ],
+    "images": [
+        # "static/description/error_message.png",
+    ],
```

Comment:
```suggestion
```

To remove

Reply:
better to add the reference to the 2 images.